packages/prebuild-config/src/plugins/__tests__/__snapshots__/withDefaultPlugins-test.ts.snap

```diff
@@ -1264,6 +1265,7 @@ Object {
                   "android:roundIcon": "@mipmap/ic_launcher_round",
                   "android:theme": "@style/AppTheme",
                   "android:usesCleartextTraffic": "true",
+                  "android:windowSoftInputMode": "adjustPan",
```

Comment:
i believe this belongs on the activity level -- MainActivity rather than MainApplication

Reply:
good catch